ImportPhotoFolders.applescript

```diff
@@ -37,7 +37,7 @@ on importFotos(aFolder, albumName, parentFolder)
 	set fotoAlbum to createFotoAlbum(albumName, parentFolder)
 
 	tell application "Photos"
-		with timeout of 600 seconds
+		with timeout of (30 * 60) seconds
 			import imageList into fotoAlbum skip check duplicates no
```

Comment:
+ application "Finder" to set theFiles to every file of aFolder whose name extension is